### ISSUE TITLE
Logic Model

### ISSUE THREAD

**Issue by MoustyHub** [MoustyHub/TSCDigitization#30]
As a (role)
I want to (...)
In order to (....)

Acceptance Criteria:....

**Comment by joshrossiter** [comment]
https://drive.google.com/file/d/1Tsrc2hIsLYWnDPuboqo90to6i8z9lEH8/view?usp=sharing

**Comment by MoustyHub** [comment]
The model looks pretty sharp, and I think it is ready for team circulation! :)

@joshrossiter:  I have a few questions on the Logic Model.

1. We seem to have several project visions circulating around - Concept Case, Problem Statements and Logic Model.  Can we align to a single one?  
Also, Are the future state characteristics definitions aligned to the ones we will prioritize against?

2. Are these indicators all being currently measured within the existing program?  There is a large quantity for us to measure and track.  A few seems more like solutions "introduction of bias reduction measures", "employee guidance in place" rather than measures.

3. I would be interested to see how this ties to the Service Inventory Team and standards for Cost Recovery Services.  Let's have a chat with Heather MacKeen's team! :) ex: remittance, etc...

**Comment by joshrossiter** [comment]
Hi Melanie,

1.a) The Project Vision that we used here is the same as what was used in the Needs Statements/Problem statements documentation. There is still some discussion to be had with the team, namely Caron, but between Lisa C, Maggie, Lisa S, and myself, this is the version we are working with.  

1.b) The Concept Case didn't articulate a vision, rather a strategic problem statement, and high level future state characteristics.  I've added those characteristics to the Logic Model, but a better approach would likely be integrating the Problem Statements once we have a complete set of working drafts. We can 'test' our thinking there to make sure we reflect the 'future state characteristics' in both the problem statements, and logic model outcomes.

2. As we were going through the key program components and outcomes, we wanted to reflect potential progress for items that may have "less direct" linkages with core metrics/KPI (really to reflect potential team progress on these items). I'm happy to adjust here as it's not a traditional approach to logic model/outcomes/measurement.

3. Sounds good!
